unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -54,14 +54,11 @@ public void Initialize()
 
             DataStore.i.debugConfig.isDebugMode.OnChange += OnDebugModeSet;
 
-            SetupDeferredRunners();
+            // SetupDeferredRunners(); // TODO: is this needed ?? (WebGL/Desktop appears to run OK without it)
```

Comment:
@Kinerius from what I tested (WebGL + Desktop) everything works fine without having this call, do you know if we really need this? if I can remove it I can also remove more code associated to that.

Reply:
It was needed in the past when this was the cornerstone of the SDK6, I think the old MessageQueue system has to go